src/core/infrastructure/migrations/1717128371946-migrations.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Migrations1717128371946 implements MigrationInterface {
```

Comment:
These migration files need names describing what they do. See how the first one is called `init`?